frontend/src/app/components/chat/Input.jsx

```diff
@@ -5,12 +5,21 @@ import Image from 'next/image';
 import { createClientComponentClient } from '@supabase/auth-helpers-nextjs';
 import { useChatStore } from '@/store/store';
 
+const ChatAvatars = [
+	"/icons/avatar-0.svg",
+	"/icons/avatar-1.svg",
+	"/icons/avatar-2.svg"
+];
+
 function Input() {
 	const baseUrl = process.env.NEXT_PUBLIC_BASE_URL;
 	const [inputValue, setInputValue] = useState('');
 	const supabase = createClientComponentClient();
 	const selectedChatId = useChatStore((state) => state.selectedChatId);
 	const user = useChatStore((state) => state.user);
+	const conversations = useChatStore((state) => state.conversations);
+	const setConversations = useChatStore((state) => state.setConversations);
```

Comment:
I think this could have been done in a single line itself through destructuring
{conversations, setConversations, setSelectedChatId} = useChatStore() 
not sure about the implementation of usechatstore tho